app/components/workspace/nrm-actions-dropdown.tsx

```diff
@@ -74,6 +78,28 @@ export function TeamMembersActionsDropdown({
           />
         </DropdownMenuItem>
 
+        <DropdownMenuItem className="text-gray-700hover:text-gray-700 p-4 hover:bg-slate-100">
+          <ControlledActionButton
```

Comment:
We don't need to control this. NRMs are available on all plans. Only thing that is not is available in free plan is importing NRMs. But editing the names should be allowed.